### DISCUSSION TITLE
Prerelease 4 beta 4 dmx out. First channel in universe seem to not work

### DISCUSSION THREAD

**Discussion by sonofice80** [forkineye/ESPixelStick#501] [General]
Hi tnx for your time and effort in making this.
I have connected a dmx Fixture to the ESP32 with prerelease 4 beta 4. If i set dmx on Fixture to say adresse 200. And set the first channel in universe to 200 and channel count to 20 in espixelstick UI i dont recive any signal. But  if i set Channel count to 200 or more i recive dmx. If i set channel count to 199 i loose dmx again.

hope i managed to explain myself.
 Tnx again for you work and time
Andreas Holm

**Comment by MartinMueller2003** [comment]
I think this is working as designed. You are mixing two settings and confusing yourself. The short answer is the E1.31 on the input section is decoupled from the DMX on the output section.

Long Answer:
On the input side you have said "For universe #1 please ignore the first 200 channels and take channel 201 and map it to output channel #1. Then continue to process received data for a total of 20 channels" That means that the input section is only receiving 20 channels of data.

Next you told the DMX output to output 20 channels of data starting at output channel #1 and ending at output channel #20. 

So when you tell your DMX fixture to listen for data at DMX channel 200, there is nothing there. You never gave that many channels to the ESP and it has no channels to send to your fixture.
